components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -320,19 +320,19 @@ class BraveRewardsBrowserTest
                                                      "/",
                                                      base::TRIM_WHITESPACE,
                                                      base::SPLIT_WANT_ALL);
-    if (url.find(braveledger_bat_helper::buildURL(REGISTER_PERSONA, PREFIX_V2,
-      braveledger_bat_helper::SERVER_TYPES::LEDGER)) == 0
-      && tmp.size() == 6) {
+    const std::string persona_url =
+        braveledger_request_util::BuildUrl(REGISTER_PERSONA, PREFIX_V2);
+    if (url.find(persona_url) == 0 && tmp.size() == 6) {
       *response = brave_test_resp::registrarVK_;
     } else if (URLMatches(url, REGISTER_PERSONA, PREFIX_V2,
-                          SERVER_TYPES::LEDGER) &&
+                          ServerTypes::LEDGER) &&
```

Comment:
This formatting looks a little off based on what I've been shown, it should be more like; 

`
    } else if (URLMatches(url, REGISTER_PERSONA, PREFIX_V2,
        ServerTypes::LEDGER) && tmp.size() == 7) {
`
?

Reply:
I just changed prefix here, don't want to introduced new changes so would leave it like this if ok with you?